client/components/themenew/hooks/use-is-fse-active.js

```diff
@@ -0,0 +1,13 @@
+import { useActiveThemeQuery } from 'calypso/data/themes/use-active-theme-query';
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+export default function useIsFSEActive() {
```

Comment:
Would naming this `useIsBlockTheme` be clearer? It seems unlikely that we're going to have FSE on a non-block theme so there's no purpose in abstracting how it's implemented, the API returns `is_block_theme` and the PHP equivalent is `wp_is_block_theme`.